modules/ListView.js

```diff
@@ -283,9 +286,17 @@
       } else if (this.emptyTemplate) {
         injectionSite.replaceWith(this.emptyTemplate(this.prepareEmpty()));
       }
-      this.trigger('render-before-dom-replacement', newDOM);
       this.$el.html(newDOM.contents());
-      this.delegateEvents();
+    },
```

Comment:
maybe rename to __addDOMTo$el() or something.  It isn't clear from the method name that it updates this.$el.  I was expecting it to return the dom fragment (but it makes sense to include the add since it is different between list and view).